lib/class-wp-theme-json-gutenberg.php

```diff
@@ -3646,4 +3646,67 @@ private static function resolve_custom_css_format( $tree ) {
 
 		return $tree;
 	}
+
+	/**
+	 * Replaces CSS variables with their values in place.
+	 *
+	 * @since 6.3.0
+	 * @param array $styles CSS declarations to convert.
+	 * @param array $values key => value pairs to use for replacement.
+	 * @return array
+	 */
+	private static function convert_variables_to_value( $styles, $values ) {
+		foreach ( $styles as $key => $style ) {
+			if ( is_array( $style ) ) {
+				$styles[ $key ] = self::convert_variables_to_value( $style, $values );
```

Comment:
We could explicitly add `continue` after this is processed, instead of letting the continuing happening later because it's not a string.